js/blocks/loader/editor.scss

```diff
@@ -104,8 +104,7 @@ div[class^="wp-block-block-lab-"],
       .bl-image__remove {
         display: inline-block;
         vertical-align: top;
-        margin-right: 0.6rem;
-        margin-top: 0.6rem;
+        margin: 1rem 0.6rem 0.6rem 0;
```

Comment:
`1rem` of margin-top might be a little too much in the Inspector. Though even the `0.6rem` that I added before was probably too much.

<img width="501" alt="space-above" src="https://user-images.githubusercontent.com/4063887/55531276-d7483f80-5666-11e9-9b8c-ef5d5ec25198.png">